authenticator/src/main/kotlin/com/bitwarden/authenticator/data/platform/datasource/disk/SettingsDiskSourceImpl.kt

```diff
@@ -138,13 +138,13 @@ class SettingsDiskSourceImpl(
             .onSubscription { emit(getBoolean(CRASH_LOGGING_ENABLED_KEY)) }
 
     override var hasUserDismissedDownloadBitwardenCard: Boolean?
-        get() = getBoolean(HAS_USER_DISMISSED_DOWNLOAD_BITWARDEN_KEY, null)
+        get() = getBoolean(HAS_USER_DISMISSED_DOWNLOAD_BITWARDEN_KEY)
```

Comment:
`:authenticator` version of `getInt` and `getBoolean` accepted a `default` parameter that is not present in the `:app` version. This updates the `:authenticator` references to use null-coalescing for default value fallback instead of passing in the default value.